src/physics/material/detail/Utils.cc

```diff
@@ -9,19 +9,68 @@
 
 #include <algorithm>
 #include <cmath>
+#include "base/Algorithms.hh"
 #include "base/Constants.hh"
+#include "base/Range.hh"
 #include "base/Quantity.hh"
 #include "physics/material/ElementDef.hh"
 
 namespace celeritas
 {
 namespace detail
 {
+//---------------------------------------------------------------------------//
+/*!
+ * Calculate Coulomb correction factor (unitless).
+ *
+ * This uses the formulation of arXiv:1204.3675 [hep-ph] of the correction
+ * factor presented in section 33.4.2 of the Review of Particle Physics. The
+ * calculation of the values is in the celeritas-docs repo at
+ * https://github.com/celeritas-project/celeritas-docs/blob/master/notes/coulomb-correction/eval.py
+ *
+ * This is accurate to 16 digits of precision through Z=92, as compared to 5
+ * or 6 digits for the equation that appears in RPP.
+ */
+real_type calc_coulomb_correction(int atomic_number)
+{
+    REQUIRE(atomic_number > 0);
+    using constants::alpha_fine_structure;
+
+    const real_type alphazsq = ipow<2>(alpha_fine_structure * atomic_number);
+
+    static const double zeta[] = {2.0205690315959429e-01,
+                                  3.6927755143369927e-02,
+                                  8.3492773819228271e-03,
+                                  2.0083928260822143e-03,
+                                  4.9418860411946453e-04,
+                                  1.2271334757848915e-04,
+                                  3.0588236307020493e-05,
+                                  7.6371976378997626e-06,
+                                  1.9082127165539390e-06,
+                                  4.7693298678780645e-07,
+                                  1.1921992596531106e-07,
+                                  2.9803503514652279e-08,
+                                  7.4507117898354301e-09,
+                                  1.8626597235130491e-09,
+                                  4.6566290650337837e-10,
+                                  1.1641550172700519e-10};
+    real_type           fz     = 1 / (1 + alphazsq);
+    real_type           azpow  = 1;
+    for (double zeta_i : zeta)
+    {
+        fz += azpow * zeta_i;
+        azpow *= -alphazsq;
+    }
+
+    ENSURE(fz > 0);
+    return alphazsq * fz;
+}
+
 //---------------------------------------------------------------------------//
 /*!
  * Calculate inverse of Tsai radiation length for bremsstrahlung [cm^2/g].
  *
- * See \S33.4.2, pp.452--453, from Review of Particle Physics, calculation of
+ * See S33.4.2, pp.452--453, from Review of Particle Physics, calculation of
```

Comment:
```suggestion
* See Review of Particle Physics [Phys. Rev. D 98, 030001 (2018)], S33.4.2, calculation of
```


Since the PDG is updated every couple years, I suggest adding its reference. I believe this comes from the previous version. In the current one, Prog. Theor. Exp. Phys. 2020, 083C01 (2020), radiation length is at S34.4.2.

Reply:
Ooh, I didn't know about the latest version. I'll download it and update there.